diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/IdentifierNaming.kt

```diff
@@ -347,11 +348,14 @@ class IdentifierNaming(private val configRules: List<RulesConfig>) : Rule("ident
         // check for methods that return Boolean
         val functionReturnType = node.findChildAfter(VALUE_PARAMETER_LIST, TYPE_REFERENCE)?.text
 
-        // if function has Boolean return type in 99% of cases it is much better to name it with isXXX or hasXXX prefix
-        if (functionReturnType != null && functionReturnType == PrimitiveType.BOOLEAN.typeName.asString()) {
-            if (BOOLEAN_METHOD_PREFIXES.none { functionName.text.startsWith(it) }) {
-                FUNCTION_BOOLEAN_PREFIX.warnAndFix(configRules, emitWarn, isFixMode, functionName.text, functionName.startOffset, functionName) {
-                    // FixMe: add agressive autofix for this
+        // We don't need to ask subclasses to rename superclass methods
+        if (node.treeParent.findChildByType(MODIFIER_LIST)?.findChildByType(OVERRIDE_KEYWORD) == null) {
```

Comment:
```suggestion
        if (node.treeParent.isOverriden()) {
```
We have a special extension function in `org.cqfn.diktat.ruleset.utils`.

Reply:
ok, it's done
on my local machine I wasn't been able to build current version. have an error `Redeclaration: AstNodeUtilsTest` when try to run `verify` task, that's why it took me so many commits🙃 Any idea why this is happening, for the future?

Reply:
No, I've never encountered redeclaration errors :) FWIW, sometimes clean build (`mvn clean install`) or invalidating caches in IDEA helps.